src/components/views/elements/DesktopCapturerSourcePicker.tsx

```diff
@@ -85,6 +85,15 @@ export default class DesktopCapturerSourcePicker extends React.Component<
     }
 
     componentDidMount() {
+        // setInterval() first waits and then executes, therefore
+        // we call getDesktopCapturerSources() here without any delay.
+        // Otherwise the dialog would be left empty for some time.
+        getDesktopCapturerSources().then((result) => {
+            this.setState({
+                sources: result,
+            });
+        });
```

Comment:
can we use async/await here just to match the style of what it looks like within the setInterval too, `componentDidMount` should be fine to be async

Reply:
I didn't think that was possible. I'll do that

Reply:
I sure hope it is :P
![image](https://user-images.githubusercontent.com/2403652/107362996-6f1fdb80-6ad1-11eb-98da-9d3d08db7e36.png)